crates/plugins/mediator-coordination/src/web/mod.rs

```diff
@@ -34,21 +33,22 @@ pub struct AppState {
     public_domain: String,
 
     // Crypto identity
-    diddoc: Document,
+    pub diddoc: Document,
     assertion_jwk: (String, Jwk),
 
     // DIDComm Resolvers
-    did_resolver: LocalDIDResolver,
-    secrets_resolver: LocalSecretsResolver,
+    pub did_resolver: LocalDIDResolver,
+    pub secrets_resolver: LocalSecretsResolver,
 
     // Persistence layer
-    repository: Option<AppStateRepository>,
+    pub repository: Option<AppStateRepository>,
```

Comment:
Did you really have to make these public?

Reply:
yes, if not i would'nt have been able to use them

Reply:
Consider using `pub(crate)` or `pub(super)` instead. This limits the visibility of the items to within the crate or module, respectively.

Reply:
:thumpsup: